web-console/4.0-designs/code-editors/code-editor.md

```diff
@@ -0,0 +1,63 @@
+# Code Editor
+
+## Edit YAML
+
+![pic 1](img/1.png)
+
+- The code area is borderless at the top and has the button bar at the bottom as a separator.
+- The Edit action can be removed from the Actions dropdown on every detail page. Users can access the tab as needed.
```

Comment:
Don't forget, we just removed this from the actions documentation as well! AKA keeping the edit action in the dropdown for now